README.md

```diff
@@ -1,54 +1,314 @@
+[![Ubuntu (Launchpad) | bizhawk](https://img.shields.io/badge/Ubuntu_(Launchpad)-bizhawk-%23E95420.svg?colorA=772953&logo=ubuntu&style=popout)](https://example.com/bizhawk)
+
+If you run `EmuHawkMono.sh` from a terminal, note that `File > Exit (Alt+F4)` doesn't terminate the process correctly, you'll need to send SIGINT (`^C`). The systems that currently work are: Game Boy + GBC (GBHawk), NES (NesHawk), Master System, Atari 7800, Commodore 64, ColecoVision, IntelliVision, TurboGrafx, and ZX Spectrum. See #1430 for progress.
+
+Is your distro not there? Released binaries can be found right here on GitHub (same download as for Windows):
+
+[![Misc. Linux | binaries](https://img.shields.io/badge/Misc._Linux-binaries-%23FCC624.svg?logo=linux&logoColor=black&style=popout)](https://github.com/TASVideos/BizHawk/releases/latest)
+
+If you download BizHawk this way, **don't mix different versions**, keep each version in its own folder. Run `EmuHawkMono.sh` to give Mono the library and executable paths — you can run it from anywhere, so putting it in a .desktop file is fine. If running the script doesn't start EmuHawk, you may need to edit it (if you use a terminal, it will say so in the output).
+
+Linux distros are supported if the distributor is still supporting your version, you're using Linux 4.4/4.9/4.14/4.19 LTS or 4.20 for x86_64/amd64, and there are no updates available in your package manager. *Please* update and reboot.
+
+macOS is supported from 10.11 "El Capitan" (Darwin 15.6). Apple doesn't seem to care about lifecycles, so we'll go with 6 months from the last security update.
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+## Building
+
+### Windows 7/8.1/10
+
+If you have WSL, Git BASH, or similar, clone the repo with:
+```
+git clone https://github.com/TASVideos/BizHawk.git BizHawk_master
+# or ssh: git clone git@github.com:TASVideos/BizHawk.git BizHawk_master
+```
+...or use a [Git GUI](https://desktop.github.com). Otherwise, you'll have to download an archive from GitHub.
+
+~~On Windows 10, open a PowerShell window in BizHawk_master (Shift+Mouse2 in File Explorer) and run `C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe /p:Configuration=Release BizHawk.sln`. On older versions, a similar Command Prompt script should work.~~ (Not sure what the right command is. Do let us know.)
```

Comment:
The easiest way to build on windows is by going to `bizhawk\Dist` and executing `BuildAndPackage_Release.bat`.

Reply:
Done. We should have the Linux build script in there too, I've been putting it in `output` so Git ignores it. See Asnivor#1.

Reply:
This won't put the build to the output folder. Output folder is used why building things from IDE. This batch puts the resulting zip right into Dist.

Reply:
fixed